cmd/gitops/ui/run/cmd.go

```diff
@@ -152,6 +152,11 @@ func runCmd(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to create cacher: %w", err)
 	}
 
+	if options.NotificationControllerAddress == "" {
+		namespace, _ := cmd.Parent().Flags().GetString("namespace")
```

Comment:
This won't work. This command doesn't have a parent which has this I believe? @foot Can I get this namespace from somewhere?